RELEASING.md

```diff
@@ -12,6 +12,9 @@ Run the following to actually upload to cargo:
 cargo publish
 ```
 
+Next you need to manually make the release in github from the tag. This will kick off the building process
+to build all the releases assets and store them on the release in github.
```

Comment:
would mention something here about pressing the button to auto-generate release notes, noticed we've forgotten to do that a couple times

Reply:
Oh I think I didn't even know this was an option. How would you do that?

Reply:
It's a button in the GitHub UI when you're creating a release. If you try to edit any of the existing releases you'll see it above the Release Notes text box